README.md

```diff
@@ -26,11 +26,10 @@ Quinn was created and is maintained by by Dirkjan Ochtman and Benjamin Saunders.
 
 ## Features
 
-* Currently targets QUIC draft 11, but with final TLS 1.3 per RFC 8446
```

Comment:
Shouldn't we mention this somewhere? A reasonable person might expect interop we don't currently have.

Reply:
It's still there a few lines lower down, under Status.